app/models/queue_item.rb

```diff
@@ -0,0 +1,19 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+
+  validates_uniqueness_of :video_id,
+    scope: :user_id
+
+  delegate :category, to: :video
+  delegate :title, to: :video, prefix: :video
+
+  def category_name
+    video.category.name
+  end
+
+  def rating
+    review = video.reviews.where(user: user).first
```

Comment:
`where(*).first` is not suggested way of searching for a single record in Rails 4 - Read the note below in this assignment - https://www.gotealeaf.com/lessons/a3a6fbb9/assignments/1566